lib/ldaputil/certmap.c

```diff
@@ -964,14 +1042,22 @@ AddAVAToBuf(char *buf, int size, int *len, const char *tagName, CERTAVA *ava)
     memcpy(buf, tagName, taglen);
     buf[taglen++] = '=';
 
-    rv = CERT_RFC1485_EscapeAndQuote(buf + taglen,
-                                     size - taglen,
-                                     (char *)ava->value.data + lenLen,
-                                     ava->value.len - lenLen);
-
-    *len += strlen(buf);
-
-    return (rv == SECSuccess ? LDAPU_SUCCESS : LDAPU_FAILED);
+    if (is_filter != 0) {
```

Comment:
Not a nitpick but curiosity. :)
Why `if (is_filter != 0) {` instead of `if (is_filter) {`?

Reply:
It is explicitly mentioned in the "Secure coding practice" 

https://cyber.gouv.fr/sites/default/files/2022/04/anssi-guide-rules_for_secure_c_language_software_development-v1.4.pdf
10.6 Implicit comparison with 0 prohibited